tom_queue.gemspec

```diff
@@ -1,4 +1,4 @@
-# coding: utf-8
+require File.expand_path('../lib/tom_queue/version', __FILE__)
```

Comment:
I'd prefer us go with require_relative or `__dir__` and drop the `".."` to leaving it. Even `File.expand_path("lib/tom_queue/version", File.dirname(__FILE__))` would be nicer than applying `".."` against the filename. (This last one is the common way of writing path expansion fwiw.)